test/elixir_tic_tac_toe_basic_test.exs

```diff
@@ -12,19 +12,30 @@ defmodule ElixirTicTacToeBasicTest do
     end
   end
 
+  defmodule TestPlayer do
+    def move(state) do
+      Map.put(state, :board, "updated board")
+    end
+  end
+
   describe "#start" do
     test "it starts the game" do
       config =
         ElixirTicTacToeBasic.start(%ElixirTicTacToeBasic{
-          ui: TestUI
+          ui: TestUI,
+          player: TestPlayer
         })
 
-      events = Map.get(config, :events, []) |> Enum.reverse()
+      %{events: events, board: board} = config
+      events = Enum.reverse(events)
 
       assert events == [
                "welcome",
+               "display board",
```

Comment:
**question**: I would have thought there would be an event to test that a player moved. Is there a gap in our test?